### PR TITLE
clean and format

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/node/error-handling/wrapping-errors-in-node-js-using-node-verror.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/node/error-handling/wrapping-errors-in-node-js-using-node-verror.md?ref=5d705d17b15231e5b2d69a04f20b933954f346b2)

File: javascript/node/error-handling/wrapping-errors-in-node-js-using-node-verror.md [hunk 1]
```diff
@@ -1,33 +1,17 @@
...
   - '[github.com](https://github.com/davepacheco/node-verror){website}'
 
-
 aspects:
   - introduction
   - workout
 
-
 ---
 # Wrapping errors in Node.js using _node-verror_
```
Comment (kapnobatai137) [inline javascript/node/error-handling/wrapping-errors-in-node-js-using-node-verror.md]:
```suggestion
# Wrapping Errors in Node.Js Using `node-verror`
```

File: javascript/node/error-handling/wrapping-errors-in-node-js-using-node-verror.md [hunk 2]
```diff
@@ -1,33 +1,17 @@
...
-  javascript.identify-common-design-patterns.5: 10
-  javascript.use-modules.2: 10
-  javascript.use-open-source-javascript-packages.6: 10
-
-tags:
-  - introduction
-  - workout
-
 links:
   - '[github.com](https://github.com/davepacheco/node-verror){website}'
```
Comment (kapnobatai137) [inline javascript/node/error-handling/wrapping-errors-in-node-js-using-node-verror.md]:
```suggestion
  - '[node-verror](https://github.com/davepacheco/node-verror){documentation}'
```

File: javascript/node/error-handling/wrapping-errors-in-node-js-using-node-verror.md [hunk 3]
```diff
@@ -39,27 +23,39 @@ The module `node-verror` can be used to produce useful error messages in Node.js
 **VError class**
```
Comment (kapnobatai137) [inline javascript/node/error-handling/wrapping-errors-in-node-js-using-node-verror.md]:
```suggestion
### `VError` Class
```

File: javascript/node/error-handling/wrapping-errors-in-node-js-using-node-verror.md [hunk 4]
```diff
@@ -39,27 +23,39 @@ The module `node-verror` can be used to produce useful error messages in Node.js
...
 ```
+
 This gives the message:
+
 ```bash
 request failed: failed to open "eg.txt":
 file not found
 ```
+
 **WError class**
```
Comment (kapnobatai137) [inline javascript/node/error-handling/wrapping-errors-in-node-js-using-node-verror.md]:
```suggestion
### `WError` Class
```

**Pull request merged**